contracts/src/registry/EnhancedAccessControl.sol

```diff
@@ -61,6 +74,15 @@ abstract contract EnhancedAccessControl is Context, ERC165 {
         _;
     }
 
+    /**
+     * @dev Constructor.
+     *
+     * @param _superAdmin The account to grant the `SUPERUSER` roles to.
+     */
+    constructor(address _superAdmin) {
+        _grantRoles(ROOT_RESOURCE, ROLE_SUPERUSER, _superAdmin);
+    }
```

Comment:
I think how you had it previously was better - we should let implementers assign whatever default permissions they want in the constructor, rather than restricting them to granting superuser access to exactly one account.